src/partner/add-listing.html

```diff
@@ -0,0 +1,231 @@
+<!DOCTYPE html>
```

Comment:
There's just so much to comment (regarding how it was structured in general, and how you named your classes) that I won't bother commenting.

Take a look at how I structured my HTML in the src/user/user-cart.html.
Take a look as well how I named the CSS classes. How the elements are grouped together, and how the CSS class names make sense together.